pallets/nft-manager/src/lib.rs

```diff
@@ -774,6 +774,35 @@ pub mod pallet {
 
             Ok(())
         }
+
+        #[pallet::weight(T::DbWeight::get().reads_writes(1 as u64, 2 as u64))]
+        #[pallet::call_index(11)]
+        pub fn sanitise_nft_data(origin: OriginFor<T>, nft_ids: Vec<U256>) -> DispatchResult {
+            let _sender = ensure_root(origin)?;
+            let bound: usize = <NftExternalRefBound as sp_core::Get<u32>>::get() as usize;
+            for nft_id in nft_ids.iter() {
+                if let Some(nft) = Self::nfts(nft_id) {
+                    if nft.unique_external_ref.len() <= bound {
+                        log::info!("NFT with id: {} was not found", nft_id);
```

Comment:
Should we update the error message. The nft is found but the external ref length is not over the bound.

Reply:
Yeah Alex raised that as well. It has been updated on the latest version